### PR TITLE
complete a few function in user32

### PR DESCRIPTION
**Pull request by varding**

### CODE REVIEW

**Comment by retep998** [comment]
Please, use spaces!

File: src/winuser.rs [hunk 1]
```diff
@@ -827,3 +827,70 @@ pub struct HELPINFO {
     pub MousePos: ::POINT,
 }
 pub type LPHELPINFO = *mut HELPINFO;
+
+//RedrawWindow() flags
+pub const RDW_INVALIDATE        : ::UINT=0x0001;
```
Comment (retep998) [inline src/winuser.rs]: Please don't vertically align things.

File: src/winuser.rs [hunk 2]
```diff
@@ -827,3 +827,70 @@ pub struct HELPINFO {
...
+    pub CtlType         : ::UINT,
+    pub CtlID           : ::UINT,
+    pub hwndItem        : ::HWND,
+    pub itemID1         : ::UINT,
+    pub itemData1       : ::ULONG_PTR,
+    pub itemID2         : ::UINT,
+    pub itemData2       : ::ULONG_PTR,
+    pub dwLocaleId      : ::DWORD,
+}
+pub type LPCOMPAREITEMSTRUCT = *mut COMPAREITEMSTRUCT;
```
Comment (retep998) [inline src/winuser.rs]: Missing end of file newline.

**Comment by varding** [comment]
@retep998 
You mean space between ":" and parameter type?

**Comment by retep998** [comment]
`pub fn CheckDlgButton(hDlg:HWND,nIDButton:c_int,uCheck:UINT)->BOOL;`
should be
`pub fn CheckDlgButton(hDlg: HWND, nIDButton: c_int, uCheck: UINT) -> BOOL;`

File: src/winuser.rs [hunk 3]
```diff
@@ -827,3 +827,70 @@ pub struct HELPINFO {
...
+pub const RDW_NOINTERNALPAINT   : ::UINT=0x0010;
+pub const RDW_NOERASE           : ::UINT=0x0020;
+
+pub const RDW_NOCHILDREN        : ::UINT=0x0040;
+pub const RDW_ALLCHILDREN       : ::UINT=0x0080;
+
+pub const RDW_UPDATENOW         : ::UINT=0x0100;
+pub const RDW_ERASENOW          : ::UINT=0x0200;
+
+pub const RDW_FRAME             : ::UINT=0x0400;
```
Comment (retep998) [inline src/winuser.rs]: For example this should be
`pub const RDW_FRAME: ::UINT = 0x0400;`

File: src/winuser.rs [hunk 4]
```diff
@@ -827,3 +827,70 @@ pub struct HELPINFO {
...
+}
+pub type LPMEASUREITEMSTRUCT = *mut MEASUREITEMSTRUCT;
+
+#[repr(C)] #[derive(Clone, Copy, Debug)]
+pub struct DRAWITEMSTRUCT {
+    pub CtlType         : ::UINT,
+    pub CtlID           : ::UINT,
+    pub itemID          : ::UINT,
+    pub itemAction      : ::UINT,
+    pub itemState       : ::UINT,
```
Comment (retep998) [inline src/winuser.rs]: Another example:
`    pub itemState: ::UINT,`

File: lib/user32-sys/src/lib.rs [hunk 5]
```diff
@@ -94,7 +94,7 @@ extern "system" {
     pub fn CharUpperBuffA(lpsz: LPSTR, cchLength: DWORD) -> DWORD;
     pub fn CharUpperBuffW(lpsz: LPWSTR, cchLength: DWORD) -> DWORD;
     pub fn CharUpperW(lpsz: LPWSTR) -> LPWSTR;
-    // pub fn CheckDlgButton();
+    pub fn CheckDlgButton(hDlg: HWND,nIDButton: c_int,uCheck: UINT) -> BOOL;
```
Comment (retep998) [inline lib/user32-sys/src/lib.rs]: Spaces after commas please!

Comment (varding) [inline lib/user32-sys/src/lib.rs]: The script below use "regexp replace" to get all things done. I hope more contributors can use it to format code quickly and avoid spent too much time on this problem.And please help improve it if someone finds a bug.

(1) Spaces after commas
",([^ \r\n])"   replace by   ", \1"
(2) space between colon and type
"([^:]):([^: ])"  replace by   "\1: \2"
(3) spaces around -> 
"([^ ])->"   replace by    "\1 ->"
"->([^ ])"   replace by    "-> \1"

This was tested on notepad++ 6.7.5

**Comment by retep998** [comment]
Do you think you could squash your PR and rebase it onto the latest master?

**Pull request closed without merging**